src/main/drivers/pwm_output.h

```diff
@@ -29,6 +29,26 @@
 #endif
 
 typedef enum {
+    DSHOT_CMD_MOTOR_STOP = 0,
+    DSHOT_CMD_BEEP1,
+    DSHOT_CMD_BEEP2,
+    DSHOT_CMD_BEEP3,
+    DSHOT_CMD_BEEP4,
+    DSHOT_CMD_BEEP5,
+    DSHOT_CMD_ESC_INFO,
+    DSHOT_CMD_SPIN_ONE_WAY, 
+    DSHOT_CMD_SPIN_OTHER_WAY, 
+    DSHOT_CMD_3D_MODE_OFF,
+    DSHOT_CMD_3D_MODE_ON, 
+    DSHOT_CMD_SETTINGS_REQUEST,
+    DSHOT_CMD_SAVE_SETTINGS, 
+    DSHOT_CMD_ROTATE_NORMAL = 20, //Blheli_S only command
+    DSHOT_CMD_ROTATE_REVERSE = 21,  //Blheli_S only command
```

Comment:
Hi @ronlix.

Thanks for the clarification. I will add this later on. Can you clarify some details please:
- is there a list of ESC types?
- 'ESC sub version letter' is ASCII?

I think it is correct for the command to return the direction that is saved, and not taking the temporary reversal (21) into account. After all the flight controller sets the reversal, so it should not have to query it.

Since this info block now includes the ESC config, am I right to assume that command 11 (send ESC settings) is now unused, and will not be implemented?

Also you ok with blheli_32 using the same format to dump ESC info over telemetry? For me, from the flight controller side, doing so would be a huge advantage, since only then will it be possible to reliably get ESC config info without the user having to provide some information. If this is ok, can I propose that we use one of the unused bytes to encode firmware type?